app/parse.py

```diff
@@ -8,8 +14,38 @@
     tags: list[str]
 
 
+def get_single_quote(quote) -> Quote:
+    return Quote(
+        text=quote.find("span", itemprop="text").text,
+        author=quote.find("small").text,
+        tags=[tag.text for tag in quote.find_all("a", class_="tag")],
+    )
+
+
+def get_quotes() -> [Quote]:
+    session = requests.Session()
+    next_url = URL
+    all_quotes = []
+    while True:
+        soup = BeautifulSoup(session.get(next_url).text, "html.parser")
+        quotes = soup.find_all("div", class_="quote")
+        all_quotes.extend([get_single_quote(quote) for quote in quotes])
+        next_page = soup.find(class_="next")
+        if not next_page:
+            break
+        next_url = URL + next_page.find("a")["href"]
+    return all_quotes
+
+
 def main(output_csv_path: str) -> None:
-    pass
+    quotes = get_quotes()
+
+    with open(output_csv_path, mode="w", newline="", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        writer.writerow(["text", "author", "tags"])
+
+        for quote in quotes:
+            writer.writerow([quote.text, quote.author, quote.tags])
```

Comment:
The `tags` attribute is a list of strings, but when writing to a CSV file, it should be converted to a single string. Consider joining the tags with a separator, such as a comma, before writing them: `','.join(quote.tags)`.